wicket-util/src/test/java/org/apache/wicket/util/lang/BytesTest.java

```diff
@@ -40,10 +40,10 @@
 	@Before
 	public void before()
 	{
-		defaultLocale = Locale.getDefault();
+		defaultLocale = Locale.getDefault(Locale.Category.FORMAT);
 
 		// these tests run in US locale.
-		Locale.setDefault(Locale.US);
+		Locale.setDefault(Locale.Category.FORMAT, Locale.US);
```

Comment:
Would the test still pass without this change ?
I.e. if Category.FORMAT is not specified then does it fallback to the `generic` locale ?